Utils/Dataflow/095_datasetInfoAMI/amiDatasets.py

```diff
@@ -196,16 +219,33 @@ def change_key_names(data):
     return data
 
 
-def remove_tid(dataset):
-    """ Remove TaskID (_tidXX) part from dataset name.
+def container_name(data):
+    """ Retrieve container name from information about dataset.
+
+    The container name is extracted from dataset name by removing
+    the '_tid...' part.
 
-    As AMI GetPhysicsParamsForDataset works with containers,
-    we construct the container name from each dataset name,
-    removing the _tid{...} part
+    :param data: dataset information
+    :type data: dict
```

Comment:
All we know about `data` after this description is that it is a dict; also, according to the line above:
```
    The container name is extracted from dataset name by removing
    the '_tid...' part.
```
-- we can assume that this dict must contain the dataset name; but there's no clue to how exactly this DS name can be provided.

I suggest to specify that there is at least one requirement to the `data`: DS name is supposed to be provided in the `'datasetname'` field -- as a part of `data` description, or where the "dataset name" is mentioned.